tests/rkt_tests.go

```diff
@@ -73,8 +74,29 @@ func (d *dirDesc) cleanup() {
 	if d.dir == "" {
 		return
 	}
-	if err := os.RemoveAll(d.dir); err != nil {
-		panic(fmt.Sprintf("Failed to remove temporary %s directory %q: %s", d.desc, d.dir, err))
+	// ideally, os.Remove should be able to remove d.dir
```

Comment:
This workaround will be removed completely. Instead we'll focus on allowing a correct cleanup, by making sure all spawned processes have been shutdown.